PepperDashEssentials/PluginLoading/PluginLoading.cs

```diff
@@ -415,14 +415,15 @@ public static void LoadPlugins()
                 // Deal with any .cplz files
                 UnzipAndMoveCplzArchives();
 
-                // Load the assemblies from the loadedPlugins folder into the AppDomain
-                LoadPluginAssemblies();
+				if(Directory.Exists(_loadedPluginsDirectoryPath) {
```

Comment:
you might consider inverting this to make it a little more readable: `if(!Directory.Exists(_loadedPluginsDirectoryPath)) { Debug.Console(0,"no loaded plugins"); return;`